petition/models.py

```diff
@@ -0,0 +1,152 @@
+class Response(models.Model):
+    senator_investigation = models.BooleanField(default=False)
+    committee_formed = models.BooleanField(default=False)
+    vote_resolution = models.BooleanField(default=False)
+    vote_referendum = models.BooleanField(default=False)
+    refer_to_other = models.BooleanField(default=False)
+    
+    investigation_info = models.CharField(max_length=1000)
+    committee_info = models.CharField(max_length=1000)
+    resolution_info = models.CharField(max_length=1000)
+    referendum_info = models.CharField(max_length=1000)
+    refer_other_info = models.CharField(max_length=1000)
+
+    # If there is a senate investigation in place, set 
+    # senator_investigation to true, false otherwise
+    def set_senate(self,bool_val):
+        self.senator_investigation = bool_val
+        
+    # If there was a committee creates, set 
+    # committee_formed to true, false otherwise
+    def set_committee (self,bool_val):
+        self.committee_formed = bool_val
+
+    # If there was a resolution voted on, set 
+    # vote_resolution to true, false otherwise
+    def set_resolution (self,bool_val):
+        self.vote_resolution = bool_val
+
+    # If there was a referendum voted on, set 
+    # vote_referendum to true, false otherwise
+    def set_referendum (self,bool_val):
+        self.vote_referendum = bool_val
+
+    # If some other action was taken/ referred to another group,
+    # set refer_to_other to true, false otherwise
+    def set_other (self,bool_val):
+        self.refer_to_other = bool_val
+
+    # Add information for the senate investigation
+    def add_info_senate(self,info):
+        self.investigation_info = info
+        return self.investigation_info
+    # Add information on the committee assigned to help
+    def add_info_committee (self,info):
+        self.committee_info = info
+        return self.committee_info
+    # Add information on the resolution that was voted on
+    def add_info_resolution (self,info):
+        self.resolution_info = info
+        return self.resolution_info
+    # Add information on the referendum that wsa voted on
+    def add_info_referendum (self,info):
+        self.referendum_info = info
+        return self.referendum_info
+    # Add information on the other group that the issue was
+    # referred to
+    def add_info_other (self, info):
+        self.refer_other_info = info
+        return self.refer_other_info
+
+class Petition(models.Model):
+    title = models.CharField(max_length=200)	# Title of petition
+    description = models.CharField(max_length=4000)	# The paragraph description
+    archived = models.BooleanField(default=False)
+    hidden = models.BooleanField(default=False)
+                
+    created_date = models.DateTimeField(db_index=True, default=timezone.now)	# Files the date created
+    expected_sig = models.IntegerField()		# The expected signature to move to the next step
```

Comment:
Looking over again and this should probably take a default value.  Otherwise the check_enough_sigs function doesn't have a value to compare to.  This is a standard value across all petitions anyway